lnbits/core/services.py

```diff
@@ -801,8 +801,6 @@ async def get_balance_delta() -> BalanceDelta:
 
 
 def get_bolt11_expiry(invoice: Bolt11) -> datetime.datetime:
```

Comment:
can you remove this function altogether and use this directly instead? https://github.com/lnbits/bolt11/blob/main/bolt11/types.py#L112
```invoice.expire_date`

you should also update bolt11 library to 2.0.6 i just fixed an issue there

Reply:
Done